src/components/withWindowDimensions/WindowHeightAdjustment.js

```diff
@@ -0,0 +1,13 @@
+// Some platforms need to adjust the window height.
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    children: PropTypes.func.isRequired,
+};
+
+const WindowHeightAdjustment = props => props.children(0);
```

Comment:
What does this do? Is the first child guaranteed to be hight window height data?

Reply:
It takes a function for children and passes it an adjustment value, the default implementation passes 0 and the android one passes insets.top. I made this to avoid having isolate the platform specific bits so we don’t have to copy the whole file.

But now that I’m thinking more about it it might be simple to just have a getHeightAdjustment(insets) function that returns 0 by default and insets.top on android.

Reply:
Thanks! Yeah, that seems a bit more self explanatory to me 👍

Reply:
Agreed with the suggestion, but I am still have a question, we have defined `WindowHeightAdjustment` and `WindowHeightAdjustment.android` but only importing `WindowHeightAdjustment.android`?

Reply:
Oh I messed up the auto import, updated to use a function and fixed that.